app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user, class_name: 'user', foreign_key: 'author_id'
-  belongs_to :post, class_name: 'post', foreign_key: 'post_id'
+  belongs_to :users, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :posts, class_name: 'Post', foreign_key: 'post_id'
```

Comment:
Remove the class_name: 'Post' because post has a model

Reply:
Hi @keneogu. Noted and all solved. Thank you :+1: